Updated the webpage with all the required features

ishaanbedi opened the pull request:
Updated the webpage with all the required features including the button to add rows.
Also updated CSS file to style rows and button and included my name in the Readme file.
See screenshot:
<img width="956" alt="Screenshot 2022-01-04 at 10 30 02 AM" src="https://user-images.githubusercontent.com/39641326/148011766-6460cde9-66e0-488a-97f9-f2590a6e448a.png">

ishaanbedi committed:
Updated the webpage with all the required features

Updated the webpage with all the required features including button to add rows.
Also updated CSS file and Readme file.

rzmk commented on js/script.js:
```diff
@@ -1,5 +1,33 @@
+    var cell6 = row.insertCell(5);
+    var cell7 = row.insertCell(6);
+
+    cell1.innerHTML = document.querySelector('.assignmentName').value;
+    cell2.innerHTML = document.querySelector('.className').value;
+    cell3.innerHTML = document.querySelector('.dateinput').value;
+    cell4.innerHTML = document.querySelector('.timeinput').value;
+    cell5.innerHTML = document.querySelector('.priorityInput').value;
+    cell6.innerHTML = document.querySelector('.estimatedTimeInput').value;
+    cell7.innerHTML = document.querySelector('.status').value;
```
We'd probably like to use `innerText` rather than `innerHTML` to prevent any cross-site scripting (XSS) hacks.

```suggestion
    cell1.innerText = document.querySelector('.assignmentName').value;
    cell2.innerText = document.querySelector('.className').value;
    cell3.innerText = document.querySelector('.dateinput').value;
    cell4.innerText = document.querySelector('.timeinput').value;
    cell5.innerText = document.querySelector('.priorityInput').value;
    cell6.innerText = document.querySelector('.estimatedTimeInput').value;
    cell7.innerText = document.querySelector('.status').value;
```

rzmk reviewed (changes requested):
Check out my comment for just 1 fix. Also here's some funny stuff that could be done if we used `innerHTML` that we probably don't want in this todo list 😆:

![image](https://user-images.githubusercontent.com/30333942/148016178-2c531da7-e999-4512-b69c-d954bee441ab.png)

ishaanbedi committed:
Updated as per the requested changes

Changed the innerHTML to innerText & applied if condition to check if the user is passing any of the input empty.

ishaanbedi commented:
Hi! Resolved the issue!

rzmk reviewed (approved):
Works great! 😄

ishaanbedi commented:
Thanks! Great working with you!
